internal/entities/rpc.go

```diff
@@ -49,7 +56,7 @@ type Transaction struct {
 	ResultXDR           string    `json:"resultXdr"`
 	ResultMetaXDR       string    `json:"resultMetaXdr"`
 	Ledger              int64     `json:"ledger"`
-	DiagnosticEventsXDR string    `json:"diagnosticEventsXdr"`
+	DiagnosticEventsXDR []string  `json:"diagnosticEventsXdr"`
```

Comment:
why is this an array now?

Reply:
Ummm, this has always been an array since the time I added the `GetTransactions` endpoint - https://github.com/stellar/stellar-rpc/pull/174/files#diff-5276db8001730f32b267cab637b316958b69aef6f8184343338855fdc118b700R66

I was actually confused when I got a parsing error in wallet-backend when it was just a `string`. And also the fact that it has been working fine before too.

Reply:
Makes sense